addons/mass_mailing_themes/views/mass_mailing_themes_templates.xml

```diff
@@ -885,4 +909,744 @@
+                            <a style="margin-left:10px" aria-label="Twitter" target="_blank" title="Twitter" href="https://twitter.com/Odoo"><span class="fa fa-twitter text-o-color-4"></span></a>  &amp;nbsp;&amp;nbsp;
+                            <a style="margin-left:10px" aria-label="Instagram" title="Instagram" target="_blank" href="https://www.instagram.com/explore/tags/odoo/"><span class="fa fa-instagram text-o-color-4"></span></a>
+                        </div>
+                        <div class="o_mail_footer_links">
+                            <a role="button" class="text-o-color-4" href="/unsubscribe_from_list">Unsubscribe</a> | <a role="button" class="text-o-color-4" href="/contactus">Contact</a>
+                        </div>
+                        <p class="o_mail_footer_copy text-o-color-4">
+                            © <t t-esc="datetime.datetime.now().year"/> All Rights Reserved
+                        </p>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </template>
+
+    <!-- Theme "Newsletter" default template -->
+    <template id="theme_newsletter_template">
+        <style id="design-element">
+            h1 {
+                color: #CE0000;
+                font-size: 54px;
+            }
+            h2{
+                font-weight: bolder;
+            }
+            h3 {
+                font-size: 16px;
+                font-weight: bolder;
+            }
+            p, p > *, li, li > * {
+                font-size: 16px;
+            }
+            a:not(.btn), a.btn.btn-link {
+                text-decoration-line: underline;
+                color: #212529;
+            }
+            a.btn.btn-primary, a.btn.btn-outline-primary, a.btn.btn-fill-primary {
+                background-color: #CE0000;
+                border-color: #CE0000;
+            }
+            a.btn.btn-secondary, a.btn.btn-outline-secondary, a.btn.btn-fill-secondary {
+                color: #CE0000;
+                background-color: #FFFFFF;
+                border-color: #CE0000;
+            }
+        </style>
+        <div class="o_snippet_view_in_browser o_mail_snippet_general pt16 pb0" style="text-align: left; padding-left: 15px; padding-right: 15px; color: rgb(183, 191, 199) !important" data-snippet="s_mail_block_header_view" data-name="View Online"><a href="/view" class="" target="_blank" data-bs-original-title="" title="">View Online</a> - <span style="color: rgb(183, 191, 199); font-size: 12.25px" t-out="datetime.datetime.now().strftime('%B %d %Y')"></span></div>
+        <div class="s_title o_mail_snippet_general pt32 pb32" data-snippet="s_title" data-name="Title" style="background-color: rgb(255, 255, 255) !important;">
+            <div class="container s_allow_columns">
+                <h1 style="text-align:center" class="">Weekly Musing</h1>
```

Comment:
Wouldn't you rather say "musings"? @edi-odoo ?

Reply:
Agreed @Zinston !